src/util/useId.ts

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+let id = 0;
+function genId() {
+  return ++id;
+}
+
+console.warn(
```

Comment:
I'd put the `console.warn` behind a `process.env.NODE_ENV !== 'production'` check.

Would need to verify that rollup is removing it at build time.